authors/apps/articles/views.py

```diff
@@ -270,4 +275,95 @@ def delete(self, request, pk, format=None):
         self.check_object_permissions(request, rating)
         rating.delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
-    
\ No newline at end of file
+    
+    
+class FavoriteView(generics.CreateAPIView):
+    """This class creates a new favorite"""
+    permission_classes = (IsAuthenticated,)
+    serializer_class = FavoriteSerializer
+
+    def post(self, request, slug):
+        """Creates a favorite"""
+        data = request.data
+        article = Articles.objects.filter(
+            # slug=slug, published=True, activated=True).first()
+            slug=slug).first()
+
+        if article is None:
+            not_found = {
+                "detail": "This article has not been found."
```

Comment:
The key should be `errors`, instead of `detail`,  as was agreed upon for error responses by the team

Reply:
Noted